### ISSUE TITLE
I am fairly certain this fossil belongs to the dinosaur clade "Thyreophora" and not to the fly genus "Thyreophora"

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#2865]
**I am fairly certain this fossil belongs to the dinosaur clade &quot;Thyreophora&quot; and not to the fly genus &quot;Thyreophora&quot;**

-----

User: [See in registry](https://www.gbif.org/api/feedback/user/4fba7c096f54def80b558ed6b38f22ed:d3cc4805baffcdce12b75f740d33a05dfcc24cf087116d083093a76fc5ce2cca270590d0c53a9929bea97798a35adfc91afefa450928c1214457de7a68fc6b0b)
System: Firefox 77.0.0 / Windows 10.0.0
Referer: https://www.gbif.org/occurrence/1900181380
Window size: width 1920 - height 966
[API log](http://elk.gbif.org:5601/app/kibana?#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2020-07-02T19:39:27.160Z',mode:absolute,to:'2020-07-02T19:45:27.160Z'))&_a=(columns:!(_source),index:'prod-varnish-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499')),sort:!('@timestamp',desc)))
[Site log](http://elk.gbif.org:5601/app/kibana?#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2020-07-02T19:39:27.160Z',mode:absolute,to:'2020-07-02T19:45:27.160Z'))&_a=(columns:!(_source),index:'prod-portal-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499')),sort:!('@timestamp',desc)))
System health at time of feedback: OPERATIONAL
datasetKey: 2814930a-3b93-48d6-b887-00979481ac9e
publishingOrgKey: 75642970-f855-11dd-8235-b8a03c50a862

**Comment by ahahn-gbif** [comment]
This seems to be a typo at source: the verbatim scientific name already reads "Thyreophora". Need to contact the publisher.
"

**Comment by mdoering** [comment]
Yes, agree. The palaeo dataset has various animal groups, so tagging the entire dataset as belonging to dinosaurs unfortunately does not work, it needs to be done on the record level. 

Otherwise we need to add a "Thyreophora" clade to the backbone, but that takes time I'm afraid

**Comment by ahahn-gbif** [comment]
email contact via helpdesk

**Comment by mnhn-paul** [comment]
Thanks for reporting this issue, we are looking into the problem.

we are looking into how to publishing the higher taxonomy for the records for this data-set. The main issue with this however is that the "working taxonomic list" that we currently use is not up to date and we might run into other issues if we publish it as is at the moment.

we will try to work our way down from the top, adding Phylum to the records might be feasible in the coming months.
